### PR TITLE
Fix bug with render models (Five Eyes model for example)

### PR DESCRIPTION
**Pull request by AndreiEvstratenko**
With this fix we able to render our "Five Eyes" models

### CODE REVIEW

**Comment by ox** [comment]
What does this PR fix specifically?

**Comment by brentchow** [comment]
@ox It's _supposed_ to match https://github.com/magicien/GLTFSceneKit/pull/28/files but it does not. 

@AndreiEvstratenko please provide more description in your PRs. Otherwise no one will know what this is for. For example, if we come back to this PR 5 months from now to figure out what changed and why, we will not remember what your intent was.

File: Source/Common/GLTFUnarchiver.swift [hunk 1]
```diff
@@ -1298,20 +1298,32 @@ public class GLTFUnarchiver {
...
+                for j in 0..<values.count {
+                    v.append(values[j])
+                }
+                animations[i].values = v
+            }
+        } else {
+            let step = animations.count
+            let dataLength = values.count / step
+            // Comment out to fix animations
+//            guard dataLength == keyTimes.count else {
```
Comment (ox) [inline Source/Common/GLTFUnarchiver.swift]: nit: indentation

Comment (AndreiEvstratenko) [reply Source/Common/GLTFUnarchiver.swift]: done

Comment (brentchow) [reply Source/Common/GLTFUnarchiver.swift]: @ox meant that these comments should start in the same column  as the `let` above

```suggestion
            // guard dataLength == keyTimes.count else {
```

Comment (AndreiEvstratenko) [reply Source/Common/GLTFUnarchiver.swift]: I commented it with special Xcode command. I think it more better

Comment (brentchow) [reply Source/Common/GLTFUnarchiver.swift]: please explain

Comment (AndreiEvstratenko) [reply Source/Common/GLTFUnarchiver.swift]: I didn't write "//" manually. I used Xcode hotkey. When you use this hotkey Xcode puts slashes on start of line (hotkey: CMD+/)

**Pull request merged**